accounts-db/src/accounts_db.rs

```diff
@@ -1627,6 +1630,10 @@ pub struct AccountsDb {
     /// (For R&D only)
     pub verify_experimental_accumulator_hash: bool,
 
+    /// Flag to indicate if the experimental accounts lattice hash is used for snapshots.
+    /// (For R&D only; a feature-gate also exists to turn this on.)
+    pub snapshots_use_experimental_accumulator_hash: AtomicBool,
```

Comment:
The reason for it to be `AtomicBool` instead of `bool` is for interior mutability for testing, not for concurrency. 
Is it right?